woocommerce/woocommerce-ios
# Edit Products: enabled right/left swipe on Product Images

Fixes #2403 

## Description
Following the behavior of Android, I added the right/left swipe when a product image is opened full screen. Now there is a new view controller `ProductImagesGalleryViewController` that substitutes the old one, and that allows us to present all the images of a product in a sequence.

## Testing

#### Test case 1
- Open a product with images.
- Tap a product image from the "Photos" screen.
- The new gallery opens and the selected image will be shown.
- Try to rotate the screen. The selected image should remain visible on the screen.
- Try to swipe left/right between images, and try to delete one or more images. Save the product and make sure that the images will be removed from the product.


#### Test case 2
- Open a product with only one image, and try to delete it from the new gallery. 
- The gallery will be dismissed.


## GIF
<img src="https://user-images.githubusercontent.com/495617/88909803-daa00b00-d25b-11ea-883b-10661293e41b.gif" width=300 />



Update release notes:
